src/transformations/simplify.jl

```diff
@@ -108,28 +98,39 @@ GI.npoint(simple)
 6
 ```
 """
+simplify(alg::SimplifyAlg, data; kw...) = _simplify(alg, data; kw...)
+# Default algorithm is DouglasPeucker
 simplify(data; calc_extent=false, threaded=false, crs=nothing, kw...) =
     _simplify(DouglasPeucker(; kw...), data; calc_extent, threaded, crs)
-simplify(alg::SimplifyAlg, data; kw...) = _simplify(alg, data; kw...)
 
-function _simplify(alg::SimplifyAlg, data; kw...)
-    ## Apply simplication to all curves, multipoints, and points,
-    ## reconstructing everything else around them.
-    simplifier(geom) = _simplify(trait(geom), alg, geom)
-    apply(simplifier, Union{PolygonTrait,AbstractCurveTrait,MultiPoint,PointTrait}, data; kw...)
-end
+#= For each algorithm, apply simplication to all curves, multipoints, and
+points, reconstructing everything else around them. =#
+_simplify(alg::SimplifyAlg, data; kw...) =
```

Comment:
No, I just use equals when it is just one function equal to another. But I see in the BlueSky style guide (new to me!) that that isn't preferred unless it fits on one line.

I guess my logic is, is we are literally just renaming the function, just like in lines 101 and 103 and those were written the exact same way.

Reply:
Yeah style guides are never quite what you want but it can simplifie some things in collaborations like this. There's a more common standard julia stule guide somewhere too Im easy either way.

Reply:
This is not so concise:
https://docs.julialang.org/en/v1/manual/style-guide/

But I think its encoded in JuliaFormatter more precisely as the default:
https://domluna.github.io/JuliaFormatter.jl/stable/style/